### PR TITLE
[PLATFORM-422]: [Localauth0] Implement a way to set permissions for audience at startup

### PR DESCRIPTION
**Pull request by cpiemontese**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-422

This PR introduces the possibility to define some starting permissions/audience mappings to facilitate setup.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
 name = "localauth0"
 version = "0.2.1"
-edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-actix-web = "4.0.0-rc.1"
 actix-files = "0.6"
+actix-web = "4.0.0-rc.1"
```
Comment (fabuloso) [inline Cargo.toml:10]: possiamo usare la 4.0.1?

File: Cargo.toml [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
 base64-url = "^1.4"
+openssl = "^0.10"
 
-chrono = { version = "^0.4", features = ["serde"] }
+chrono = {version = "^0.4", features = ["serde"]}
 
 thiserror = "^1.0"
 
 jsonwebtoken = "8.0.0-beta.6"
+moka = "^0.7"
```
Comment (fabuloso) [inline Cargo.toml:25]: possiamo rimuovere i ^ che in rust non servono?

**Review by fabuloso — DISMISSED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -1,25 +1,31 @@
...
+actix-web = "4.0.1"
+
+derive-getters = "0.2.0"
+
+serde = {version = "1.0.137", features = ["derive"]}
+serde_json = "1.0.80"
 
-serde = { version = "^1.0", features = ["derive"] }
-serde_json = "^1.0"
+base64-url = "1.4.13"
```
Comment (cottinisimone) [inline Cargo.toml:17]: Order nazi strike again :)

**Review by fabuloso — APPROVED** [review]

**Review by cottinisimone — CHANGES_REQUESTED** [review]

File: Makefile.toml [hunk 4]
```diff
@@ -1,28 +1,31 @@
+[env]
+LOCALAUTH0_CONFIG_PATH = "/home/app/localauth0.toml"
```
Comment (cottinisimone) [inline Makefile.toml:2]: I think this file should be retrieved from outside the container

**Review by cottinisimone — COMMENTED** [review]
I think we need to test if the env variable works as expected, maybe building the image and configuring it with that env var

File: Makefile.toml (hunk 4 above)
Comment (cpiemontese) [reply Makefile.toml:2]: So `./localhost.toml`? Or maybe even `localhost.toml` if it works.

**Review by fabuloso — DISMISSED** [review]

**Pull request merged**